### PR TITLE
Add cam alvium

### PR DESCRIPTION
**Pull request by TitouanGragnic**
refacto error handling of camera_alvium

### CODE REVIEW

File: Camera/CameraAlvium/camera_alvium.cc [hunk 1]
```diff
@@ -8,6 +8,9 @@
 #include <span>
 namespace camera
 {
+
+#define VMB_ERROR(cond) cond != VmbErrorType::VmbErrorSuccess
```
Comment (GustaveHerve) [inline Camera/CameraAlvium/camera_alvium.cc:12]: Maybe add parenthesis around cond and the whole macro for consistency and safety

Comment (GustaveHerve) [inline Camera/CameraAlvium/camera_alvium.cc:12]: Maybe add parenthesis around cond and the whole macro for consistency and expansion safety

Comment (alexispinson) [reply Camera/CameraAlvium/camera_alvium.cc:12]: I also think this might be a good thing to do

Comment (ArthurCourselle) [reply Camera/CameraAlvium/camera_alvium.cc:12]: I also think this might be a good thing to do

Comment (ArthurCourselle) [reply Camera/CameraAlvium/camera_alvium.cc:12]: But you reviewed in 1 minutem are you sure that you read all the code correclty ???

Comment (TitouanGragnic) [reply Camera/CameraAlvium/camera_alvium.cc:12]: gnngngn ( ok i do it)

**Pull request merged**